packages/common-server/src/stateValidator.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  DendronError,
+  DEngineClient,
+  ErrorFactory,
+  RespV3,
+  TreeUtils,
+  VaultUtils,
+} from "@dendronhq/common-all";
+import path from "path";
+import { getAllFiles } from "./files";
+import { vault2Path } from "./filesv2";
+
+/**
+ * Validate correctness of engine notes compared with filesystem.
+ * Currently supports parent/children relationship between notes
+ */
+export class StateValidator {
```

Comment:
nit: to follow the new conventions, can you name this file with capital `StateValidator.ts`?

[[Conventions|dendron://dendron.docs/dev.process.code#conventions]]